engine/app/components/citizens_advice_components/header.html.haml

```diff
@@ -9,12 +9,15 @@
       .cads-grid-col-md-2.cads-header__logo-row
         = logo
         - if search_form.present?
-          = tag.button(class: "cads-header__search-reveal js-cads-search-reveal cads-icon_search",
-            title: t(".open_search"),
-            "aria-expanded": "false",
```

Comment:
I think you need to keep `aria-expanded`, looking at the docs: https://w3c.github.io/aria/#aria-expanded

Reply:
Although saying that I'm explicitly setting the other aria attributes in the template so you're right I should explicitly leave this for clarity.